WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -243,6 +247,8 @@ - (instancetype)initWithPost:(AbstractPost *)post
             
             _ownsPost = YES;
         }
+        
+        _sharingController = [[PostSharingController alloc] init];
```

Comment:
Probably no need to create this during in it since its not likely to be used very often.  There's probably no need to save a reference in a property either.  Maybe just create / recreate on demand.